lib/KModal.vue

```diff
@@ -261,6 +261,8 @@
       // if modal contains KSelect, special classes & styles will be applied
       const kSelectCheck = document.querySelector('div.modal div.ui-select');
       this.containsKSelect = !!kSelectCheck;
+      this.observeContentChanges();
```

Comment:
Hmm I dont think we need to apply more listeners, since with the current implementation we already catch the changes, and run the `updateContentSectionStyle` method on every update. If we remove these listeners the solution should continue working.